src/applications/ivc-champva/10-7959C/config/submitTransformer.js

```diff
@@ -30,20 +30,17 @@ export default function transformForSubmit(formConfig, form) {
       copyOfData.applicantName?.middle?.charAt(0) ?? '';
   }
 
+  // Get today's date as YYYY-MM-DD
+  copyOfData.certificationDate = new Date().toISOString().replace(/T.*/, '');
+
   // Make sure all dates are in MM-DD-YYYY format
   Object.keys(copyOfData).forEach(key => {
     if (key.toLowerCase().includes('date')) {
-      copyOfData[key] = new Date(copyOfData[key])
-        // MM-DD-YYYY date w/ hyphens instead of slashes
-        .toLocaleDateString('es-pa')
-        .replace(/\//g, '-');
+      const date = copyOfData[key];
+      copyOfData[key] = `${date.slice(5)}-${date.slice(0, 4)}`;
```

Comment:
Changing form using the locale format that just happens to match what we want to a simple string re-arrangement.